common/components/common/groups/ContactGroupButton.jsx

```diff
@@ -0,0 +1,182 @@
+// @flow
+
+import React from 'react';
+import {Button} from 'react-bootstrap';
```

Comment:
For consistency with how we import react-bootstrap elsewhere, this should be `import Button from 'react-bootstrap/Button';`